### PR TITLE
chore(base): remove breaking styles from _common.scss

### LINKED ISSUE #5261: Bug - Remove outdated styles in _common.scss
**Describe the problem**

Follow up to #5218 
Towards https://github.com/patternfly/patternfly-react/pull/5275

**How do you reproduce the problem?**

Tested on a local branch with the changes in https://github.com/patternfly/patternfly-react/pull/5275. The icon sizes are correct now when we import the svg classes following the changes in #5218 . However some outdated styles in `_common.scss` are now causing the PF site to look like this when built:

Screen Shot 2022-11-16 at 3 47 32 PM

After investigating, removing these outdated styles seems to fix it. https://github.com/patternfly/patternfly/blob/main/src/patternfly/base/_common.scss#L2-L11

These are already expected to be removed in the breaking-change release based on the comments.

### PR DESCRIPTION
**Pull request by jenny-s51**
Closes #5261

### CODE REVIEW

File: src/patternfly/base/_common.scss [hunk 1]
```diff
@@ -1,26 +1,4 @@
...
-  &::before,
-  &::after {
-    padding: 0;
-    margin: 0;
-    background-color: transparent;
-  }
-}
-
-// remove in breaking change
-@if $pf-global--unset-root-font-size {
```
Comment (mcoker) [inline src/patternfly/base/_common.scss:14]: we can also remove this https://github.com/patternfly/patternfly/blob/14e45911682ee81e822cecc02c22cb380a98c05b/src/patternfly/sass-utilities/scss-variables.scss#L15

**Review by mcoker — CHANGES_REQUESTED** [review]
so nice!!! one small change to remove a sass var we no longer need.

**Review by jenny-s51 — COMMENTED** [review]
Thank you @mcoker ! Updated 🙂

**Review by mcoker — APPROVED** [review]
nice 😎

**Pull request merged**